### PR TITLE
[ONNX] minor clarifications of docstrings

### PR DESCRIPTION
**Pull request by garymm**
1. Make description of ONNX_ATEN_FALLBACK more accurate (after #67460).
2. Specify minimum and maximum values for opset_version. This is pretty
   important information and we should make users dig through source
   code to find it.

### CODE REVIEW

**Comment by garymm** [comment]
@xadupre LGTY?

File: torch/onnx/__init__.py [hunk 1]
```diff
@@ -188,18 +188,18 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
...
 
               .. warning::
 
                 Models exported this way are probably runnable only by Caffe2.
 
-        opset_version (int, default 9):
-            Must be ``== _onnx_main_opset or in _onnx_stable_opsets``,
-            defined in torch/onnx/symbolic_helper.py.
+        opset_version (int, default 9): The version of the
+            `default opset <https://github.com/onnx/onnx/blob/master/docs/Operators.md>`_
```
Comment (BowenBao) [inline torch/onnx/__init__.py:201]: what does `default opset` mean?

Comment (garymm) [reply torch/onnx/__init__.py:201]: If an op doesn't specify a domain, it's in the default opset AKA `ai.onnx`.
I guess if it's not clear to you then it definitely won't be clear to many people.
Any suggestions as to how to write this to make it clearer?

Comment (BowenBao) [reply torch/onnx/__init__.py:201]: exchanging `default` with `ai.onnx` or `ONNX` or `default (ai.onnx)`?

**Review by BowenBao — APPROVED** [review]
LGTM

**Comment by garymm** [comment]
Merging. @xadupre if this is still not clear, LMK and I will be happy to make another change.

**Pull request merged**